terraform/modules/apps/publisher/main.tf

```diff
@@ -61,15 +61,15 @@ data "aws_acm_certificate" "public_lb_alternate" {
 }
 
 resource "aws_lb" "public" {
-  name               = "fargate-public-${var.service_name}"
+  name               = "${var.service_name}-${terraform.workspace}"
```

Comment:
I wonder if perhaps in the `govuk` module we could pass in an argument called `workspace_service_name`. It might be nicer to use this variable rather than creating `"${var.service_name}-${terraform.workspace}"` in a few different places.